storage/series/series.go

```diff
@@ -162,14 +164,17 @@ func (s *dbSeries) updateBlocksWithLock() updateBlocksResult {
 			continue
 		}
 
-		if !currBlock.IsRetrieved() {
+		if cachePolicy == CacheAllMetadata && !currBlock.IsRetrieved() {
```

Comment:
Bugger, should probably deprecate this policy now.  I'll open an issue for it, maybe can spend some cycles myself too unwrapping it.

Reply:
sounds good